course_contents/2_intro_to_python/notes/01. Conditions.md

```diff
@@ -63,14 +74,19 @@ if name == best_friend:
 
 ### The `else` keyword
 - the `else` keyword is used to run a block of code if all the other `if`/`elif` statements have failed before it
-- when using the `else` keyword there shouldn't be any condition following it, only the block of code after it
-- the syntax for using the `else` keyword is simple:
-    1. write the `else` keyword in the same indent as the last `if`/`elif` statement
-    2. follow the keyword with colon
-    3. indented block of code that will get executed if other statements fail
+- the `else` keyword is followed by a colon without any condition before it
+- syntax for using the `else` keyword:
 
-An example of proper usage of the `else` keyword:
+> we can add some kind of a reminder that we can also chain elifs before the else
+```py
+if <expr>:
+    <statement>
+else:
+    <statement>
+```
 
+An example of proper usage of the `else` keyword:
+> again, is there really a need for lines before the if statement?
```

Comment:
(FWIW, instead of leaving these questions inside the code, it's easier if you make a comment in the PR yourself!)

I think instead of `input()`, we can just hard-code the string of somebody's name. Here it should trigger the `else` clause (so not `"Rolf"` or `"Phil"`).

Also something funny with the code, we've got the best friend and relative names and then we ask the user for their name. They themselves can't be a best friend or a family member, or a stranger for that matter since they are themselves!

^ just a funny thing, it won't be a problem if we hardcode the name.

Reply:
Changed this to make more sense and hardcoded names